src/InteractivePathAnalysis/CustomMenu.cpp

```diff
@@ -0,0 +1,88 @@
+#include "CustomMenu.h"

```

Comment:
For all new files those are missing: 
* Copyright header
* FOEDAG namespace

Reply:
corrected 
copyright header - https://github.com/os-fpga/FOEDAG/pull/1495/commits/14b143d83c69d1c91198d698b000d69725f46b04
FOEDAG namespace - https://github.com/os-fpga/FOEDAG/pull/1495/commits/63d56e4d5896eab69aea7e14fc82741c8ab7dcea